<launch>

  <arg name="model" default="$(find wpr1_bringup)/urdf/wpr1.urdf"/>
  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />

  <!-- wpr1 manipulator-->
  <node pkg="wpr1_bringup" type="wpr1_manipulator" name="wpr1_manipulator" output="screen">
      <param name="serial_port" type="string" value="/dev/wpr1_mani"/> 
  </node>

</launch>